src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/PosBlockSignatureRule.cs

```diff
@@ -16,6 +16,9 @@ namespace Stratis.Bitcoin.Features.Consensus.Rules.CommonRules
     [IntegrityValidationRule]
     public class PosBlockSignatureRule : StakeStoreConsensusRule
     {
+        /// <summary>When checking the POS block signature this determines the maximum push data (public key) size following the OP_RETURN in the nonspendable output.</summary>
+        private const int MaxPushDataSize = 40;
```

Comment:
Are you sure the max is 40> I was sure it was 80.

Reply:
80 is for normal op return data. 40 is for pub key here.

Reply:
You need to compare with stratisX. In C# code the relay policy is 83 bytes but that has nothing to do with consensus

Reply:
But you totally should double check with the C++ code, Not to rely on whatever reclaim

Reply:
True, nothing to do with Consensus.
I know about the MAX_OP_RETURN_RELAY in both nodes and the values are different:

In StratisX:
https://github.com/stratisproject/stratisX/blob/8dd4601d9089dd66132d127a98bfbc035e3b5aab/src/script.h#L25-L27

In the C# node: 
https://github.com/stratisproject/StratisBitcoinFullNode/blob/2993932daad9e996bef6473235f970d1e3b281ce/src/NBitcoin/StandardScriptTemplate.cs#L79-L81

@Aprogiena Can you show me the C++ code you're referring to?

> 80 is for normal op return data. 40 is for pub key here.

Can you please show me where this is in the C++ code?

Reply:
Please see CBlock::CheckBlockSignature() in C++ node.